### PR TITLE
Menu list - react

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-06 at 15 26 23

### CODE REVIEW

File: src/components/list/MenuList.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+const SIZE = {
+  SMALL: 'small',
+  NORMAL: 'normal',
+  LARGE: 'large'
+};
+
+const MenuList = ({items, size}) => {
+  const listClass = classNames('sg-menu-list', {
+    [`sg-menu-list--${size}`]: size !== SIZE.NORMAL
```
Comment (aju) [inline src/components/list/MenuList.jsx]: what happens when size is undefined ?

Comment (anna-wro) [inline src/components/list/MenuList.jsx]: nothing breaks, the list has a default size (there are only additional classes for small and large size)

Comment (aju) [inline src/components/list/MenuList.jsx]: doesn't it have --undefined class ?:)

Comment (anna-wro) [inline src/components/list/MenuList.jsx]: Whoops! I've just fixed it :)

**Comment by aju** [comment]
@anna-wro please bum version :)

**Comment by anna-wro** [comment]
@aju I did it :)

**Pull request merged**